### ISSUE TITLE
A* algorithm

### ISSUE THREAD

**Issue by AlexeyVolosnikov** [AndGem/OsmToRoadGraph#69]
Do you have A* algorithm for routing (using data which is got from osmtoroadgraph)?

**Comment by AndGem** [comment]
Hi Alexey,

thanks for creating an issue. There is currently no routing algorithm implemented in this repository. However, a simple Dijkstra algorithm should be fairly easy to implement, and I was thinking of adding for an example. 

Do you have a specific use-case in mind?

**Comment by AlexeyVolosnikov** [comment]
Yes, I know 2 main routing algorithms. I got a question with good answer [here](https://codereview.stackexchange.com/questions/235898/speeding-up-dijkstras-algorithm) about dijkstra algorithm so you could add function from AJNeufelds answer (because it is faster a little rather than my) or dijkstra function my question :). By the way I have A* algorithm with heristic, as far as I tested it it is working fine, but I am a newer and my code as far as I am presume is 'dirty' and not so readable. I can send it to you to read if you want.

**Comment by AndGem** [comment]
Hi @AlexeyVolosnikov 

sorry for the long silence. I'm working in the background on a bigger change to the project which should make using routing algorithms quite simple.

Currently, I'm working on this PR #70 
This allows to save the output of the program to a file format that can be read by [networkx](https://networkx.github.io/). There are some integrated routing algorithms, but it should also be easy to play around with your own. I'll add an example on how to do that. I'll update once this is done.
